### DISCUSSION TITLE
My laptop full not working

### DISCUSSION THREAD

**Discussion by deflikk** [valinet/ExplorerPatcher#1386] [Q&A] [answered]
After starting up, my laptop completely stopped functioning normally
![IMG20230427004256](https://user-images.githubusercontent.com/130749401/234659507-a384c206-0124-407e-ac21-4b1a12503a7e.jpg)
![IMG20230427004250](https://user-images.githubusercontent.com/130749401/234659520-70949dfd-ccd8-4031-b837-d415a47187c7.jpg)

**Comment by deflikk** [comment]
The only thing I can do is open the task manager with ctrl alt delete

**Comment by tallman69** [comment]
What version of Windows are you running?  Have you installed any Windows updates or other software lately?  What version of ExplorerPatcher are you using?

**Reply by deflikk** [reply]
Windows 11, latest version EP, didnt instal windows updates. I fixed it with cmd but my start button stopped working. Most likely because I renamed the file dxgi.dll to dxgio.dll, but if I put it back, it crashes again completely

**Comment by deflikk** [comment, marked as answer]
If you have the same problem, you can use the Task Manager to open cmd MUST BY ADMINISTRATOR NAME and rename the file dxgi.dll to dxgio.dll (ren "c:windows\dxgi.dll" dxgio.dll) but you will not have the start button
